source/agora/consensus/protocol/Nominator.d

```diff
@@ -57,6 +57,9 @@ import core.time : msecs, seconds;
 
 mixin AddLogger!();
 
+// Set a maximum number of transactions to a thousand
+enum MaxTransactionsPerBlock = 1000;
```

Comment:
This will only be a stop-gap measure. But we should really be checking the block size, not the amount of transactions.

For now let's go ahead with it though.

Reply:
I'll add a TODO to the code to indicate this.

Reply:
Updated